source/base/TimeSpan.ooc

```diff
@@ -42,28 +42,19 @@ TimeSpan: cover {
 		(minuteString, secondString, milliString) := (this toMinutes(), this toSeconds(), this toMilliseconds()) toString()
 		(dayModString, hourModString, minuteModString) := (this toDays() modulo(7), this toHours() modulo(24), this toMinutes() modulo(60)) toString()
 		(secondModString, milliModString) := (this toSeconds() modulo(60), this toMilliseconds() modulo(1000)) toString()
-		second := format replaceAll("%w", weekString)
-		result := second replaceAll("%D", dayString)
-		second free()
-		second = result replaceAll("%H", hourString)
-		result free()
-		result = second replaceAll("%M", minuteString)
-		second free()
-		second = result replaceAll("%S", secondString)
-		result free()
-		result = second replaceAll("%Z", milliString)
-		second free()
-		second = result replaceAll("%d", dayModString)
-		result free()
-		result = second replaceAll("%h", hourModString)
-		second free()
-		second = result replaceAll("%m", minuteModString)
-		result free()
-		result = second replaceAll("%s", secondModString)
-		second free()
-		second = result replaceAll("%z", milliModString)
-		(result, weekString, dayString, hourString, minuteString, secondString, milliString, dayModString, hourModString, minuteModString, secondModString, milliModString) free()
-		second
+		result := format replaceAll("%w", weekString)
+		result = result replaceAll("%D", dayString, true, true)
```

Comment:
I'd prefer to see something like
```ooc
result = result replaceAll("%D", dayString, String CaseSensitive, String FreeArgument)
```
or sth. but we can start with `true, true` ;)